c/catboost/catboost-1.2.5_ubi_9.3.sh

```diff
@@ -0,0 +1,121 @@
+BUILD_HOME=$(pwd)
+PYTHON_VERSION=3.11.5
+CMAKE_VERSION=3.28.1
+CLANG_VERSION=17.0.6
+export PATH=/usr/local/bin:/usr/bin:$PATH
+
+#Install Centos repos and dependencies
+yum config-manager --add-repo https://mirror.stream.centos.org/9-stream/CRB/ppc64le/os
+yum config-manager --add-repo https://mirror.stream.centos.org/9-stream/AppStream//ppc64le/os
+yum config-manager --add-repo https://mirror.stream.centos.org/9-stream/BaseOS/ppc64le/os
+rpm --import http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-Official
+
+yum install -y wget unzip zip git gcc-c++ zlib-devel openssl-devel libffi-devel sqlite-devel xz-devel perl ninja-build gfortran bzip2-devel xz lld libjpeg-turbo-devel openblas-devel texinfo
+
+#Install Python from source
+cd $BUILD_HOME
+if [ -z "$(ls -A $BUILD_HOME/Python-${PYTHON_VERSION})" ]; then
+    wget https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tgz
+    tar xzf Python-${PYTHON_VERSION}.tgz
+    rm -rf Python-${PYTHON_VERSION}.tgz 
+    cd Python-${PYTHON_VERSION}
+    ./configure --with-system-ffi --with-computed-gotos --enable-loadable-sqlite-extensions
+    make -j ${nproc} 
+else
+    cd Python-${PYTHON_VERSION}
+fi
+make altinstall
+ln -sf $(which python3.11) /usr/bin/python3
+ln -sf $(which pip3.11) /usr/bin/pip3
+python3 -V && pip3 -V
+	
+#Install cmake
+cd $BUILD_HOME
+if [ -z "$(ls -A $BUILD_HOME/cmake-${CMAKE_VERSION})" ]; then
+    wget -c https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}.tar.gz
+    tar -zxvf cmake-${CMAKE_VERSION}.tar.gz
+    rm -rf cmake-${CMAKE_VERSION}.tar.gz
+    cd cmake-${CMAKE_VERSION}
+    ./bootstrap --prefix=/usr/local/cmake --parallel=2 -- -DBUILD_TESTING:BOOL=OFF -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_USE_OPENSSL:BOOL=ON
+else
+    cd cmake-${CMAKE_VERSION}
+fi
+make install -j$(nproc)
+export PATH=/usr/local/cmake/bin:$PATH
+cmake --version
+
+#Setup clang
+cd $BUILD_HOME
+wget https://github.com/llvm/llvm-project/releases/download/llvmorg-$CLANG_VERSION/clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8.tar.xz
+tar -xvf clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8.tar.xz
+rm -rf clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8.tar.xz
+mv clang+llvm-$CLANG_VERSION-powerpc64le-linux-rhel-8.8 clang-$CLANG_VERSION
+export PATH=$BUILD_HOME/clang-$CLANG_VERSION/bin:$PATH
+export CC=$BUILD_HOME/clang-$CLANG_VERSION/bin/clang
+export CXX=$BUILD_HOME/clang-$CLANG_VERSION/bin/clang++
+export ASM=$BUILD_HOME/clang-$CLANG_VERSION/bin/clang
+clang --version
+
+#Clone the repository 	
+cd $BUILD_HOME
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+pip3 install six setuptools wheel jupyterlab Pillow pandas conan==1.62.0 plotly scipy testpath pytest ipywidgets 'numpy<2.0'
+conan install . || true && sed -i 's/ autotools.configure()/ autotools.configure(args=["--build=powerpc64le-linux-gnu"])/g' /$BUILD_HOME/root/.conan/data/yasm/1.3.0/_/_/export/conanfile.py
```

Comment:
Ideally, this should be 
`conan install . || true && sed -i 's/ autotools.configure()/ autotools.configure(args=["--build=powerpc64le-linux-gnu"])/g' /$BUILD_HOME/root/.conan/data/yasm/1.3.0/_/_/export/conanfile.py && **conan install .**`
because the first one failed.

Reply:
Not needed as we have only executed this to make power arch. related change in yasm code.